README.md

```diff
@@ -1,4 +1,11 @@
-# Ripple Network Crawl
+# Rippled Network Crawler
+
+This crawls the ripple network,
+via making requests to the /crawl endpoint of each peer it can connect to, starting from an entry point. 
+Some peers may know, and publish (perhaps errantly .. ), the ip associated with a peer, while others
+don't. We merge the points of view of each peer, collecting a dict of data, keyed by ip address.
+
+This maps out the connections between all rippled servers (not necessarily UNLS) who (for the most part) don't even participate in Consensus or at least don't have any say in influencing the outcome of a transaction on mainnet. 
```

Comment:
Its good style for files to have a maximum width of 80 characters. In vim you can do ``set tw=80``, then run ``V`` to select the lines and ``gq`` to have vim apply smart indentation.